spark/v3.3/spark/src/test/java/org/apache/iceberg/TestFileIOSerialization.java

```diff
@@ -101,6 +104,60 @@ public void testHadoopFileIOJavaSerialization() throws IOException, ClassNotFoun
     Assert.assertEquals("Conf values must be present", "v2", actualConf.get("k2"));
   }
 
+  @Test
```

Comment:
Why are these tests still here? If the test cases moved into individual modules, then there should be no need to re-test in Spark.

Reply:
Agree with you, i left it here as I wanted to have a test with spark e2e, but what it's testing is not different from what we have already tested in their respective modules. Have removed them.